pyproject.toml

```diff
@@ -12,9 +12,119 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 [build-system]
-requires = ["setuptools==69.5.1"]
+requires = ["setuptools==69.5.1", "wheel"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "optimum-neuron"
+dynamic = ["version"]
+description = "Optimum Neuron is the interface between the Hugging Face Transformers and Diffusers libraries and AWS Trainium and Inferentia accelerators. It provides a set of tools enabling easy model loading, training and inference on single and multiple neuron core settings for different downstream tasks."
+readme = "README.md"
+requires-python = ">=3.10"
+license = {text = "Apache-2.0"}
+authors = [
+    {name = "HuggingFace Inc. Special Ops Team", email = "hardware@huggingface.co"},
+]
+keywords = ["transformers", "diffusers", "mixed-precision training", "fine-tuning", "inference", "trainium", "inferentia", "aws"]
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Education",
+    "Intended Audience :: Science/Research",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+dependencies = [
+    "transformers ~= 4.51.0",
+    "accelerate == 0.29.2",
+    "optimum ~= 1.23.3",
+    "huggingface_hub >= 0.29.0",
+    "numpy>=1.22.2, <=1.25.2",
+    "protobuf>=3.20.3, <4",
+]
+
+[project.urls]
+Homepage = "https://huggingface.co/hardware/aws"
```

Comment:
the link is not valid anymore, maybe this? https://huggingface.co/docs/optimum-neuron/index